exoplanets/exoplanets.ipynb

```diff
@@ -344,7 +348,7 @@
    "id": "195cf044",
    "metadata": {},
    "source": [
-    "There aren't a lot of potentially habitable exoplanets, and most of them lie in the cluster detected by Kepler. We can also filter by whether the planet is potentially escapable with a chemical rocket, which leaves even fewer! We'll set their size to 30 so they're more visible."
+    "There aren't a lot of potentially habitable exoplanets, and most of them are small and lie in the cluster detected by Kepler. We can also filter by whether the planet is potentially escapable with a chemical rocket, which leaves even fewer! We'll set their size to 30 so they're more visible."
```

Comment:
```suggestion
    "There aren't a lot of potentially habitable exoplanets, and most of them are small and lie in the cluster detected by Kepler. We can also filter by whether the planet is potentially escapable with a chemical rocket, which leaves even fewer! We'll circle those points with a large purple-colored ring so that you can spot them."
```